src/API.jl

```diff
@@ -915,7 +915,7 @@ function precompile(ctx::Context)
         precomp_events[pkgid.uuid] = Base.Event()
         was_recompiled[pkgid.uuid] = false
     end
-    # TODO: since we are a complete list, but not topologically sorted, handling of recursion will be completely at random
+    
```

Comment:
@KristofferC I looked at the git blame and it seems these todos were yours. I think they're both dealt with in this PR, so removed them

Reply:
They are not mine, they come from:

https://github.com/JuliaLang/julia/pull/28402/files#diff-f472857fede6d42cd50c2de4cdb414c3R465

but Pkg was developed in Base and in a separate repo at the same time so when I synced them, they show up as mine.

Reply:
Ah ok